python/pyhusky/backend/library/functional.py

```diff
@@ -16,8 +16,8 @@
 
 import cloudpickle
 
-from pyhusky.backend.globalvar import GlobalVar, GlobalSocket, OperationParam, GlobalN2NSocket
-from pyhusky.common.serializers import Serializer
+from python.pyhusky.backend.globalvar import GlobalVar, GlobalSocket, OperationParam, GlobalN2NSocket
```

Comment:
Certainly yes for development. Then you don't need to use the installed method. A very simple way is to set env `$PYTHONPATH` like `$ export PYTHONPATH=$PYHUSKY_ROOT/python` and this is compatible with `import pyhusky.*`. Adding `python.pyhusky` is too ugly.